packages/lesswrong/components/analytics/AnalyticsGraph.tsx

```diff
@@ -37,6 +35,11 @@ const styles = (theme: ThemeType): JssStyles => ({
       marginBottom: 0,
     }
   },
+  graphHeaderSmallerTitle: {
```

Comment:
Oh this one is because it needs to be different on the author analytics page and the post analytics page (the title needs to be in that component because there is a "checking latest data..." label that shows up inline with the title